.rubocop.yml

```diff
@@ -129,6 +129,7 @@ Metrics/BlockLength:
     - 'modules/*/spec/**/*.rb'
     - 'modules/*/app/swagger/**/*.rb'
     - 'modules/*/config/routes.rb'
+    - 'modules/mobile/config/initializers/statsd.rb'
```

Comment:
Can you add this to the `.rubocop_todo.yml` instead via `rubocop --auto-gen-config`? You may have to run `rubocop -a` afterwards to correct any redundant disables.